### ISSUE TITLE
[1.8.9] Crash on...full login?

### ISSUE THREAD

**Issue by Insufficient-Charge** [SonarSonic/Calculator#113]
Latest version on Curse Forge.
I logged into my world after I added Calculator to my modpack. I crashed as soon as my fps moved above 2.
I tried again, logging into my world. I was able to pause the game before crashing again, and even adjust video settings just fine. As soon as I unpaused, I crashed.

There are many mods in my modpack. If there are any specific ones, please let me know and I will give it a try. My first look is LoginShield. >_>

I am using Minecraft 1.8.9 and Calculator 2.0.1

**Comment by AEnterprise** [comment]
a crashlog is required, without it we don't even know what crashes (if it was calculator or not and if it is what part specificaly)

also what do you mean by 'my first look is loginshield'? do you suspect that mod is causing the crash? if so why are you opening an issue here?

**Comment by SonarSonic** [comment]
Yeah I need a crash log to do anything. Also 1.8.9 is basically dead...

**Comment by Insufficient-Charge** [comment]
I was saying it might be a conflict, since uninstalling Calculator fixed
it, and login shield does stuff during login. Will get a log over when I
get a chance.

**Comment by AEnterprise** [comment]
1.8.9 might be dead but 1.8.9 -> 1.9+ is minor so the bug will almost certainly be there as well
